dll/shellext/netshell/lanconnectui.cpp

```diff
@@ -341,6 +341,13 @@ CNetConnectionPropertyUi::LANPropertiesUIDlg(
                 return PSNRET_NOERROR;
             }
 #endif
+            if (lppl->hdr.code == NM_DBLCLK)
```

Comment:
Any reason why this is using another variable (`lppl` instead of `lppsn`) as the code above?